react/__tests__/SearchFooter.test.js

```diff
@@ -1,27 +1,43 @@
 /* eslint-env jest */
-import { render } from 'test-utils'
+import { render, fireEvent } from '@vtex/test-tools/react'
 import React from 'react'
 
 import SearchFooter from '../components/SearchFooter'
 
 describe('<SearchFooter /> component', () => {
+  const getLinkProps = jest.fn().mockImplementation(pageNumber => {
+    return {
+      page: `mockedPage${pageNumber}`,
+      params: 'mockedParams',
+    }
+  })
   const renderComponent = customProps => {
     const props = {
       recordsFiltered: 20,
       page: 1,
       maxItemsPerPage: 5,
       runtime: { navigate: () => {} },
-      getLinkProps: jest.fn(),
+      getLinkProps,
       ...customProps,
     }
-    return render(<SearchFooter {...props} />)
+    return { ...render(<SearchFooter {...props} />), getLinkProps }
   }
 
   it('should be rendered', () => {
     const { asFragment } = renderComponent()
     expect(asFragment()).toBeDefined()
   })
 
+  it('should return link', () => {
+    const { container, getLinkProps } = renderComponent()
+    const mainDiv = container.querySelector('.ph2.pointer.c-on-base')
```

Comment:
Use test-id instead or query for a text.

This is really helpful:
https://testing-library.com/docs/guide-which-query#docsNav